components/purchase_modal/purchase_modal.tsx

```diff
@@ -179,7 +193,191 @@ function Card(props: CardProps) {
+                    />
+                </a>
+            </div>
+        </>
+    );
+
+    const errorMessage = formatMessage({
+        id: 'admin.billing.subscription.userCount.error',
+        defaultMessage: `Your workspace currently has ${props.usersCount} users`,
+    }, {num: props.usersCount});
+
+    const yearlyPlan = (
+        <>
+            <div className='flex-grid'>
+                <div className='user_seats_container'>
+                    <Input
+                        name='UserSeats'
+                        type='text'
+                        value={usersCount}
+                        onChange={onChange}
+                        placeholder={'User seats'}
+                        wrapperClassName='user_seats'
+                        inputClassName='user_seats'
+                        customMessage={checkGreaterThanUsersCount() ? null : {
+                            type: ItemStatus.ERROR,
+                            value: errorMessage,
+                        }}
+                    />
+                </div>
+                <div className='icon'>
+                    <OverlayTrigger
+                        delayShow={Constants.OVERLAY_TIME_DELAY}
+                        placement='right'
+                        overlay={userCountTooltip}
+                    >
+                        <i className='icon-information-outline'/>
+                    </OverlayTrigger>
+                </div>
+                <div className='monthly_price'><p>{`$${monthlyPrice}`}</p></div>
+            </div>
+            <table>
+                <tbody>
+                    <tr>
+                        <td className='yearly_savings'>
+                            <FormattedMessage
+                                defaultMessage={'Yearly Savings'}
+                                id={'admin.billing.subscription.yearlySavings'}
+                            />
+                        </td>
+                        <td className='yearly_savings'>{`-$${priceDifference}`}</td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <FormattedMessage
+                                defaultMessage={'Total'}
+                                id={'admin.billing.subscription.total'}
+                            />
+                        </td>
+                        <td>{`$${yearlyPrice}`}</td>
+                    </tr>
+                </tbody>
+            </table>
+            <div className='enable_annual_sub'>
+                <button
+                    className={'plan_action_btn ' + props.buttonDetails.customClass}
+                    disabled={props.buttonDetails.disabled}
+                    onClick={props.buttonDetails.action}
+                >{props.buttonDetails.text}</button>
+            </div>
+            <div className='plan_billing_cycle'>
+                <FormattedMessage
+                    defaultMessage={'You will be charged {beginDate}. '}
```

Comment:
This will be charged immediately so we need to change the copy here. Left a comment on Figma for updated copy @annelieseklein